### ISSUE TITLE
Phpactor does not resolve Pest's Expectation mixin

### ISSUE THREAD

**Issue by fnzr** [phpactor/phpactor#2463]
On a clean project with only pest installed, phpactor cannot find mixin functions from the Pest\Expectation class:

![2023-12-10-15:57:01-screenshot](https://github.com/phpactor/phpactor/assets/5471818/f04a7f43-b710-47bb-8567-7a23ca3d5145)

The expect function resolves into a Pest\Expectation object, declared like so:

```php
/**
 * @template TValue
 *
 * @property OppositeExpectation $not Creates the opposite expectation.
 * @property EachExpectation $each Creates an expectation on each element on the traversable value.
 * @property PendingArchExpectation $classes
 * @property PendingArchExpectation $traits
 * @property PendingArchExpectation $interfaces
 * @property PendingArchExpectation $enums
 *
 * @mixin Mixins\Expectation<TValue>
 * @mixin PendingArchExpectation
 */
final class Expectation
{
...
```

The `Mixins\Expectation<TValue>`  contains a bunch of methods that aren't recognized, like `toBe` and `toBeTrue`. Is this a configuration issue on my side?

**Comment by dantleech** [comment]
I'd imagine rather generics not being supported on Mixins.

**Comment by dantleech** [comment]
should be fixed in #2472 let me know otherwise :)
